fdbserver/BlobWorker.actor.cpp

```diff
@@ -975,33 +1021,51 @@ ACTOR Future<BlobFileIndex> compactFromBlob(Reference<BlobWorkerData> bwData,
 		}
 		ASSERT(snapshotVersion < version);
 
+		// TODO: optimization - batch 'encryption-key' lookup given the GranuleFile set is known
+		// FIXME: get cipher keys for delta as well!
```

Comment:
This fixme is fixed by this change, right?

Reply:
Yup, seems like a state TODO at the moment. Shall remove it